app/components/pull-request-page.js

```diff
@@ -67,6 +67,19 @@ export default Ember.Component.extend({
     )
   },
 
+  isVisible: Ember.computed("model.status", "session.isCommitter", function() {
+    let committer = get(this, "session.isCommitter")
```

Comment:
You don't need to change it, but in these cases, I try to keep the variable name the same as teh field name so there are fewer nouns going on.

Reply:
So, `let isCommitter = get(this, "session.isCommitter")`

Reply:
changed it anyway, in the name of best practice.